cmodel/predefined/_seirv_model.py

```diff
@@ -4,12 +4,14 @@
 
 
 class ModelSEIRV(CompartmentalModel):
+
     def build_model(
         self, t, y,
-        Lambda, mu, alpha, omega, gamma, xi1, xi2, sigma, b1, b2, b3, c1, c2, c3
+        Lambda, mu, alpha, omega, gamma, xi_E, xi_I, sigma, b1, b2, b3, c1, c2, c3
     ) -> List[float]:
```

Comment:
Suggestion, just to keep  a consistent convention between all models:

```python
    def build_model(
        self, t, y,
        Lmbd, mu, alpha, omega, gamma, xi_E, xi_I, sigma, b_E, b_I, b_V, c_E, c_I, c_V
    ) -> List[float]:
```

Note that you need to change all the occurrences of the corresponding parameters within the function.